eng/Versions.props

```diff
@@ -74,7 +74,7 @@
     <MicrosoftVisualStudioPackagesVersion>17.5.274-preview</MicrosoftVisualStudioPackagesVersion>
     <RoslynPackageVersion>4.6.0-3.23164.6</RoslynPackageVersion>
```

Comment:
This should change to be a property matching the version.details.xml element you added: https://github.com/dotnet/razor/pull/8492/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3R13

`MicrosoftCodeAnalysisCSharpVersion`

Reply:
OK - I'll introduce the property, but also keep using `RoslynPackageVersion` as it's used in many places in this file.

Reply:
@mmitche I've updated the dependency and tweaked the implementation a bit, in https://github.com/dotnet/razor/pull/8492/commits/049cc5de658642a708e1a75f6bdb8928da385cc1

I think this is now a bit cleaner.